base-server.d.ts

```diff
@@ -0,0 +1,704 @@
+import { Log, Store, ServerConnection, TestTime } from '@logux/core'
+import { Server as HTTPServer } from 'http'
+
+import { Context, ChannelContext, LoguxPatternParams } from './context'
+import { ServerClient } from './server-client'
+
+/**
+ * Logux meta
+ */
+export type LoguxMeta = {
+  id: string
+  time: number
+  subprotocol: LoguxBaseServerOptions['subprotocol']
+  server: string
+  reasons: string[]
+  channels?: string[]
+  clients?: string[]
+  status?: 'waiting' | 'add' | 'clean' | 'processed' | 'subscribed'
+}
+
+/**
```

Comment:
In Logux Core I do not comment utilities classes. I have comments only for classes.